### PR TITLE
[ticket/14086] Add mcp_forum_topic_title_* template events

### PR DESCRIPTION
**Pull request by Zoddo**
Allows to add some information before/after the topic title.

https://tracker.phpbb.com/browse/PHPBB3-14086

### CODE REVIEW

**Comment by Zoddo** [comment]
!set Event

**Comment by marc1706** [comment]
Please call these _before/_after

**Comment by Zoddo** [comment]
Same as #3766, no?

**Comment by Zoddo** [comment]
Renamed

**Comment by marc1706** [comment]
Could you please re-create the ticket in the tracker? This seems to have been lost during the database outage.

**Comment by Zoddo** [comment]
@marc1706: Done.

So, I also need to edit the commit message?

**Comment by marc1706** [comment]
Yes please.

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -504,6 +504,22 @@ mcp_ban_unban_before
 * Since: 3.1.0-RC3
 * Purpose: Add additional fields to the unban form in MCP
 
+mcp_forum_topic_title_before
+===
+* Locations:
+    + styles/prosilver/template/mcp_forum.html
+    + styles/subsilver2/template/mcp_forum.html
+* Since: 3.1.6-RC1
+* Purpose: Add some information on the left of the topic title
```
Comment (marc1706) [inline phpBB/docs/events.md]: Should say before here and after in the after event. Left or right really depends on whether you have a ltr or rtl language.

Comment (Zoddo) [inline phpBB/docs/events.md]: modified

**Pull request merged**